findbugs fails

```
$ mvn clean install -DskipTests
...
[INFO] --- findbugs-maven-plugin:3.0.2:findbugs (findbugs) @ xacml-sdk-core ---
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for AuthZForce :: XACML SDK 5.0.0-SNAPSHOT:
[INFO] 
[INFO] AuthZForce :: XACML SDK ............................ SUCCESS [  0.490 s]
[INFO] AuthZForce :: XACML SDK :: MODEL ................... SUCCESS [  1.211 s]
[INFO] AuthZForce :: XACML SDK :: COMMON .................. SUCCESS [  0.389 s]
[INFO] AuthZForce :: XACML SDK :: API ..................... SUCCESS [  0.293 s]
[INFO] AuthZForce :: XACML SDK :: CORE .................... FAILURE [  2.444 s]
[INFO] AuthZForce :: XACML SDK :: DISTRIBUTION ............ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.052 s
[INFO] Finished at: 2019-05-27T10:09:04+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:findbugs-maven-plugin:3.0.2:findbugs (findbugs) on project xacml-sdk-core: Unable to parse configuration of mojo org.codehaus.mojo:findbugs-maven-plugin:3.0.2:findbugs for parameter pluginArtifacts: Cannot assign configuration entry 'pluginArtifacts' with value '${plugin.artifacts}' of type java.util.Collections.UnmodifiableRandomAccessList to property of type java.util.ArrayList -> [Help 1]

```

with JDK 1.8.0_201 + Ubuntu 18.04 LTS

PS sorry for logging it here but  OW2 Issue Tracker is currently returning 503

--- a/.gitignore
+++ b/.gitignore
@@ -9,5 +9,3 @@
 **/*.pmdruleset.xml
 .idea/
 *.iml
-.idea/
-*.iml

--- a/.gitignore
+++ b/.gitignore
@@ -7,3 +7,7 @@
 *dependency-reduced-pom.xml
 **/*.pmd
 **/*.pmdruleset.xml
+.idea/
+*.iml
+.idea/
+*.iml

--- a/authzforce-xacmlsdk-core/pom.xml
+++ b/authzforce-xacmlsdk-core/pom.xml
@@ -124,7 +124,7 @@
 					<plugin>
 						<groupId>org.codehaus.mojo</groupId>
 						<artifactId>findbugs-maven-plugin</artifactId>
-						<version>3.0.2</version>
+						<version>3.0.4</version>
 						<configuration>
 							<!-- Enables analysis which takes more memory but finds more bugs. 
 								If you run out of memory, changes the value of the effort element to 'Low'. -->

